docs/src/content/docs/reference/helm-chart-config.mdx

```diff
@@ -0,0 +1,749 @@
+        </tr>
+    </tbody>
+</table>
+
+For production environments, these should typically be set to false. Instead, external managed databases should be used.
+
+## Services
+
+<table>
+    <thead>
+        <tr>
+            <th className='w-48'>Field</th>
+            <th>Type</th>
+            <th>Default</th>
+            <th>Description</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td>`disableWebsite`</td>
+            <td>Boolean</td>
+            <td>false</td>
+            <td>If true, disables the frontend website deployment.</td>
+        </tr>
+        <tr>
+            <td>`disableBackend`</td>
+            <td>Boolean</td>
+            <td>false</td>
+            <td>If true, disables the backend API deployment.</td>
+        </tr>
+        <tr>
+            <td>`disablePreprocessing`</td>
+            <td>Boolean</td>
+            <td>false</td>
+            <td>If true, disables preprocessing pipelines.</td>
+        </tr>
+        <tr>
+            <td>`disableIngest`</td>
+            <td>Boolean</td>
+            <td>false</td>
+            <td>If true, disables ingestion services.</td>
+        </tr>
+        <tr>
+            <td>`disableEnaSubmission`</td>
+            <td>Boolean</td>
+            <td>false</td>
+            <td>If true, disables ENA (European Nucleotide Archive) submission service.</td>
+        </tr>
+        <tr>
+            <td>`customWebsiteImage`</td>
+            <td>String</td>
+            <td></td>
+            <td>Optional custom Docker image for the website.</td>
+        </tr>
+    </tbody>
+</table>
+
+## Organism Configuration
+
+<table>
+    <thead>
+        <tr>
+            <th className='w-48'>Field</th>
+            <th>Type</th>
+            <th>Default</th>
+            <th>Description</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td>`organisms`</td>
+            <td>Object</td>
+            <td></td>
+            <td>An object where the keys are the organism IDs and values are an [Organism (type)](#organism-type)</td>
+        </tr>
+    </tbody>
+</table>
+
+### Organism (type)
+
+Each organism object has the following fields:
+
+<table>
+    <thead>
+        <tr>
+            <th className='w-48'>Field</th>
+            <th>Type</th>
+            <th>Default</th>
+            <th>Description</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td>`schema`</td>
+            <td>[Schema (type)](#schema-type)</td>
+            <td></td>
+            <td></td>
+        </tr>
+        <tr>
+            <td>`preprocessing`</td>
+            <td>A list of [Preprocessing (type)](#preprocessing-type)</td>
+            <td></td>
+            <td>Configuration for the preprocessing pipeline(s)</td>
+        </tr>
+        <tr>
+            <td>`ingest`</td>
+            <td>[Ingest (type)](#ingest-type)</td>
+            <td></td>
+            <td>Configuration for the preprocessing pipeline(s)</td>
```

Comment:
```suggestion
            <td>Configuration for the ingest pipeline(s)</td>
```
This specifies the corresponding field in NIH, and will be used to map fields ingested from NIH to Loculus fields.

Reply:
Thanks, I fixed it.